contracts/IdentityOracle.sol

```diff
@@ -83,66 +80,27 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
         _onlyOracle();
         lastStartUpdProcInvoked = block.timestamp;
         Chainlink.Request memory req = buildChainlinkRequest(
-            stringToBytes32(JOBID_GET_STATE_HASH),
+            stringToBytes32(JOBID_GET_STATE_HASH_IPFSCID),
             address(this),
-            this.setTmpStateHash.selector
+            this.setFulfillStateHashIPFSCID.selector
         );
-        sendChainlinkRequestTo(ORACLE_ADDRESS, req, ORACLE_PAYMENT);
+        requestOracleData(req, ORACLE_PAYMENT);
```

Comment:
i'm not sure  I understand this. why does the oracle need to start a request to itself??
1. if there's a cronjob then oracle simply puts the merklehash every  time cronjob  is run. and calls  setState.
2. if someone wants to trigger a merklehash refresh before the cronjob then it makes sense that anyone can call this method with payment, and it will trigger the request to the oracle